packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 2.2.1 - July 26 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
+
+New Feature:
+- Add CLI command 'teamsfx preview' to directly preview Teams apps after scaffolding.
+- For multiple developers collaborate on one project, Teams Toolkit enable developers to create their own local environment by pressing F5. 
+- Add encryption to secret data in .userdata file, and support view/edit secret data through VS Code UI and CLI tool.
+
+Enhancement:
```

Comment:
Keep consistent with previous style to use `Added` and `Enhanced`?

Reply:
I would prefer this one since ‘new feature’ looks more formal than ‘Added’? haha, we propose to use it all afterwards.